R/utilities-task.R

```diff
@@ -136,7 +140,7 @@ loadCalculateSensitivityTask <- function(workflow, active = FALSE, settings = NU
 
   return(SensitivityAnalysisTask$new(
     getTaskResults = runSensitivity,
-    nameTaskResults = deparse(substitute(runSensitivity)),
+    nameTaskResults = getObjectNameAsString(runSensitivity),

```

Comment:
reads better. You could even simply by getName, extractName etc.. But at least we know what's going on. great